manifests/03-cluster-operator.yaml

```diff
@@ -5,3 +5,9 @@ apiVersion: config.openshift.io/v1
 kind: ClusterOperator
 metadata:
   name: ingress
+status:
+  versions:
+    - name: operator
+      version: "0.0.1-snapshot"
+    - name: ingress-controller
+      version: "0.0.1-snapshot"
```

Comment:
[The CVO documentation](https://github.com/openshift/cluster-version-operator/blob/8402d219f36fc79e03edf45918785376113f2cc1/docs/dev/clusteroperator.md#what-should-be-the-contents-of-clusteroperator-custom-resource-in-manifests) says only to add an entry for the operator itself.

Reply:
Fixed